tasks/configure-RedHat.yml

```diff
@@ -0,0 +1,15 @@
+- name: Install orientdb as a service
+  template: >-
```

Comment:
I'd prefer the ":" notation to keep a coherent syntax with the rest of the playbook

```
template:
    src: orientdb.service.j2
    dest: /usr/lib/systemd/system/orientdb.service
    owner: root
    group: root
    mode: 400
```